### PR TITLE
Update to Rails 6.1 and new cookie policy

### PR DESCRIPTION
**Pull request by khcr**
Update to Rails 6.1 which includes a new cookies policy which removes the warning in Firefox.

### CODE REVIEW

File: bin/rails [hunk 1]
```diff
@@ -1,9 +1,5 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path('../spring', __FILE__)
-rescue LoadError => e
```
Comment (nkcr) [inline bin/rails:4]: is there a reason to remove the rescue ?

File: config/environments/production.rb [hunk 2]
```diff
@@ -46,9 +48,9 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # config.force_ssl = true
 
-  # Use the lowest log level to ensure availability of diagnostic information
-  # when problems arise.
-  config.log_level = :debug
+  # Include generic and useful information about system operation, but avoid logging too much
+  # information to avoid inadvertent exposure of personally identifiable information (PII).
+  config.log_level = :info
```
Comment (nkcr) [inline config/environments/production.rb:53]: wondering if it won't log too much information. `:warn` could be better.

File: config/initializers/filter_parameter_logging.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```
Comment (nkcr) [inline config/initializers/filter_parameter_logging.rb:5]: good to know!

**Review by nkcr — APPROVED** [review]
Minor comment. To be ignored if this is just new generated files.

File: bin/rails (hunk 1 above)
Comment (khcr) [reply bin/rails:4]: no, that is a default file.

File: config/environments/production.rb (hunk 2 above)
Comment (khcr) [reply config/environments/production.rb:53]: That is also a default, but `warn` is an option.

**Pull request merged**